src/core/agent/services/signifyNotificationService.ts

```diff
@@ -102,6 +102,24 @@ class SignifyNotificationService extends AgentService {
         notifications.notes.shift();
       }
       for (const notif of notifications.notes) {
+        if (notif.a.r === NotificationRoute.MultiSigIcp) {
+          const multisigNotification = await this.signifyClient
+            .groups()
+            .getRequest(notif.a.d);
+          const multisigId = multisigNotification[0]?.exn?.a?.gid;
+          if (!multisigId) {
+            continue;
```

Comment:
Won't we constantly have this in our loop? We should mark the notification as read